aten/src/ATen/native/native_functions.yaml

```diff
@@ -5521,16 +5521,22 @@
     CPU: _symeig_helper_cpu
     CUDA: _symeig_helper_cuda
 
+- func: _eig_helper(Tensor self, bool eigenvectors) -> (Tensor, Tensor)
```

Comment:
looks like this function is just a helper that does not need dispatch and does not need to be exposed to python, so it does not need to be in native_functions.yaml (like apply_eig)